docker/deploy.sh

```diff
@@ -78,6 +86,13 @@ run_docker_swarm() {
             sleep 1
             exit 1
         fi
+        if ! docker stack deploy -c $docker_file -c $gw_docker_file op-scim
+        then
+            echo " "
+            echo "Failed to deploy to Docker Swarm; investigate the error before proceeding"
+            sleep 1
+            exit 1
+        fi
     fi
 
     if ! docker stack deploy -c $docker_file op-scim
```

Comment:
I wonder if this is necessary? Is this not covered by the `docker stack deploy` on line 67?